### PR TITLE
Export EmphasizeElementProps from common, deprecate from frontend.

### PR DESCRIPTION
**Pull request by bbastings**
Requested for saved views client implementation to not require frontend dependencies.

### CODE REVIEW

File: common/api/imodeljs-common.api.md [hunk 1]
```diff
@@ -2326,6 +2336,26 @@ export interface ElementProps extends EntityProps {
     userLabel?: string;
 }
 
+// @public
+export interface EmphasizeElementsProps {
+    // (undocumented)
+    alwaysDrawn?: Id64Array;
```
Comment (pmconne) [inline common/api/imodeljs-common.api.md:2342]: `@public` properties should be documented. For "props" types like this it often suffices to put, for example, `@see [EmphasizeElements.unanimatedAppearance]($frontend)`.

**Review by pmconne — APPROVED** [review]
Dunno if you feel like addressing the lack of documentation.

**Review by bsteinbk — APPROVED** [review]

**Review by kabentley — APPROVED** [review]

File: common/api/imodeljs-common.api.md (hunk 1 above)
Comment (bbastings) [reply common/api/imodeljs-common.api.md:2342]: Sounds good.

File: core/frontend/src/EmphasizeElementsProps.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Rendering
+ */
+
+import { AppearanceOverrideProps as CommonAppearanceOverrideProps, EmphasizeElementsProps as CommonEmphasizeElementsProps, FeatureOverrideType as CommonFeatureOverrideType } from "@bentley/imodeljs-common";
```
Comment (aruniverse) [inline core/frontend/src/EmphasizeElementsProps.ts:9]: Just curious/nit picking here. Did you need to alias the imports to re-export them?

ie, could you have done this?
```
import {  FeatureOverrideType } from "@bentley/imodeljs-common";
/** Options for overriding element appearance.
 * @deprecated use [FeatureOverrideType]($common)
 * @public
 */
export FeatureOverrideType;
```

**Review by aruniverse — APPROVED** [review]

**Review by diegopinate — APPROVED** [review]

File: core/frontend/src/EmphasizeElementsProps.ts (hunk 2 above)
Comment (bbastings) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: export FeatureOverrideType; <- not valid.

Comment (pmconne) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: He's saying: don't do import `as CommonXyz` you could just import and export `Xyz` directly.

Comment (bbastings) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: Yeah, I get that...you get stuff like this trying to do that:

Parsing error: Declaration or statement expected.

Comment (bbastings) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: If someone wants to mess with this, disable the auto-merge and go ahead, I'm eating lunch.

Comment (pmconne) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: too late. :)

Comment (pmconne) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: This works, but you lose the ability to deprecate/document them.
```
import { AppearanceOverrideProps, EmphasizeElementProps, FeatureOverrideType } from "@bentley/imodeljs-common";
export { AppearanceOverrideProps, EmphasizeElementProps, FeatureOverrideType };
```

Comment (bbastings) [reply core/frontend/src/EmphasizeElementsProps.ts:9]: Yes, likely also triggers the error about exports w/o release tags...

**Pull request merged**